crates/sui-indexer/migrations_v2/2023-08-19-044035_tx_indices/up.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE tx_indices (
+    tx_sequence_number          BIGINT       PRIMARY KEY,
+    transaction_digest          bytea        NOT NULL,
+    input_objects               bytea[]      NOT NULL,
+    changed_objects             bytea[]      NOT NULL,
+    senders                     bytea[]      NOT NULL,
```

Comment:
each tx only has one sender, do we need an array here?

Reply:
in the future we may have multiple senders, so making this future proof